Neos.ContentRepositoryRegistry/Classes/ContentRepositoryRegistry.php

```diff
@@ -275,6 +278,28 @@ private function buildProjectionsFactory(ContentRepositoryId $contentRepositoryI
         return $projectionsAndCatchUpHooksFactory;
     }
 
+    /** @param array<string, mixed> $contentRepositorySettings */
+    private function buildCommandHooksFactory(ContentRepositoryId $contentRepositoryId, array $contentRepositorySettings): CommandHooksFactory
+    {
+        $commandHooksSettings = $contentRepositorySettings['commandHooks'] ?? [];
+        if (!is_array($commandHooksSettings)) {
+            throw InvalidConfigurationException::fromMessage('Content repository "%s" does not have the "commandHooks" configured properly. Expected array, got %s.', $contentRepositoryId->value, get_debug_type($commandHooksSettings));
+        }
+        $commandHookFactories = [];
+        foreach ((new PositionalArraySorter($commandHooksSettings))->toArray() as $name => $commandHookSettings) {
```

Comment:
I added support for explicit ordering of hooks because e.g. a logging hook should always be executed last